src/info/code_size.rs

```diff
@@ -9,30 +8,30 @@ use crate::{
 
 #[derive(Serialize)]
 #[serde(rename_all = "camelCase")]
-pub struct LocInfo {
-    pub lines_of_code: usize,
+pub struct CodeSizeInfo {
+    pub size: usize,
     #[serde(skip_serializing)]
     number_separator: NumberSeparator,
 }
 
-impl LocInfo {
-    pub fn new(loc_by_language: &[(Language, usize)], number_separator: NumberSeparator) -> Self {
-        let lines_of_code = get_total_loc(loc_by_language);
+impl CodeSizeInfo {
+    pub fn new(size_by_language: &[(Language, usize)], number_separator: NumberSeparator) -> Self {
+        let size = size_by_language.iter().map(|(_, size)| size).sum();
         Self {
-            lines_of_code,
+            size,
             number_separator,
         }
     }
 }
 
 #[typetag::serialize]
-impl InfoField for LocInfo {
+impl InfoField for CodeSizeInfo {
     fn value(&self) -> String {
-        format_number(&self.lines_of_code, self.number_separator)
+        format_number(&self.size, self.number_separator)
```

Comment:
Is there a unit? Bytes?

Reply:
Bytes (assuming that's the unit gix returns)

cc @Byron